docs/request.md

```diff
@@ -0,0 +1,104 @@
+# BLE Request Proof Function Documentation
+
+## Overview
+
+The `bleRequestProof` function is designed to facilitate a proof request over Bluetooth Low Energy (BLE).
+
+## Function Signature
+
+```typescript
+export const bleRequestProof = async (options: BleRequestProofOptions) => Promise<string>
+```
+
+## Parameters
+
+The function takes a single object parameter of type `BleRequestProofOptions` with the following properties:
+
+- `agent: Agent` - An instance of the Agent class from the Credo framework.
+- `peripheral: Peripheral` - A Peripheral object representing the BLE device.
+    - This can be retrieved with the `usePeripheral` hook as shown in the example.
+- `serviceUuid: string` - The UUID of the BLE service to use.
+    - This UUID needs to be established out-of-band, i.e. by generating one and scanning a QR code.
+- `presentationTemplate: PresentationTemplate` - An object containing the proof request details.
+- `onFailure: () => Promise<void> | void` - A callback function to be called if the proof request fails.
+- `onConnected?: () => Promise<void> | void` - An optional callback function to be called when the BLE connection is established.
+- `onDisconnected?: () => Promise<void> | void` - An optional callback function to be called when the BLE connection is disconnected.
+
+## Return Value
+
+Proof record id
+
+## Usage Example
+
+```typescript
+import { Agent } from '@credo-ts/core';
+import { bleRequestProof, PresentationTemplate, usePeripheral } from '@animo-id/react-native-ble-didcomm';
+import { YourPeripheralImplementation } from './your-peripheral-implementation';
```

Comment:
I don't see this being used, so it's not clear what `YourPeripheralImplementation` should do?